### PR TITLE
Add p2p sent messages logging

### PR DESCRIPTION
**Pull request by anjmao**
Some more logging for p2p messages.

### CODE REVIEW

File: core/connection/manager.go [hunk 1]
```diff
@@ -384,17 +384,17 @@ func (manager *connectionManager) createP2PSession(c Connection, p2pChannel p2p.
...
 			PaymentVersion: string(session.PaymentVersionV3),
 		},
 		ProposalID: int64(proposal.ID),
 		Config:     config,
-	})
-
-	res, err := p2pChannel.Send(p2p.TopicSessionCreate, sessionRequest)
+	}
+	log.Info().Msgf("Sending P2P message to %q: %s", p2p.TopicSessionCreate, sessionRequest.String())
+	res, err := p2pChannel.Send(p2p.TopicSessionCreate, p2p.ProtoMessage(sessionRequest))
```
Comment (Waldz) [inline core/connection/manager.go]: Please do logging inside `p2pChannel.Send()`, will be hard to remember to add this liners before each sending or receiving.
Also code will be less noisy.

Comment (anjmao) [reply core/connection/manager.go]: Send message accepts this struct
```go
type Message struct {
	Data []byte
}
```
It is not coupled to concrete message struct, you just send bytes. If I add it to Send I can't log formatted payload with fields. There are only 5 places there it is needed right now and we will not have much more of these messages.

Also, what if you want to add more context to logging?  I suggest to keep it explicit as is.

Comment (Waldz) [reply core/connection/manager.go]: Well serialisation should happen in transport level also :)

File: session/pingpong/exchange_messaging.go [hunk 2]
```diff
@@ -64,8 +65,9 @@ func (es *ExchangeSender) Send(em crypto.ExchangeMessage) error {
 		AgreementTotal: em.AgreementTotal,
 		Provider:       em.Provider,
 		Signature:      em.Signature,
-	})
-	_, err := es.ch.Send(p2p.TopicPaymentMessage, pMessage)
+	}
+	log.Info().Msgf("Sending P2P message to %q: %s", p2p.TopicPaymentMessage, pMessage.String())
```
Comment (Waldz) [inline session/pingpong/exchange_messaging.go]: I suppose all these struct dumps should not be in info level

Comment (anjmao) [reply session/pingpong/exchange_messaging.go]: Changed to debug level.

File: core/connection/manager.go (hunk 1 above)
Comment (anjmao) [reply core/connection/manager.go]: p2p transport layer works with raw bytes, there is no need to for extra abstractions.

**Pull request merged**